packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/LineGraph/Reflow/SimplePanning.1.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import LineGraphPanning from '../../../example/Graphs/LineGraph/Reflow/SimplePanning?dev-site-example';
+import PropsTable from '@cerner/terra-graphs/lib/components/Line/LineGraph?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Line/lineTimeseriesPanning.js?dev-site-codeblock';
+import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Line/simpleLineData.js?dev-site-codeblock';
```

Comment:
Change dataset to one we use in examples https://github.com/cerner/terra-graphs/pull/152/files#diff-22bf74e1f8ccbadbf683519900ec4ca5301a788d1f337cb370360d507bcafb1bR7